emperor/support_files/js/util.js

```diff
@@ -13,8 +13,12 @@ define(['underscore'], function(_) {
     var numericPart = [], alphaPart = [], result = [];
 
     // separate the numeric and the alpha elements of the array
+    // Note that NaN and +/- Infinity are not considered numeric elements
     for (var index = 0; index < list.length; index++) {
-      if (isNaN(parseFloat(list[index]))) {
+      if (
+          isNaN(parseFloat(list[index])) ||
```

Comment:
Worth doing the parse to float a single time?

Reply:
yeah, would be nice (and should make this if statement fit on one line i guess). will resolve tomorrow